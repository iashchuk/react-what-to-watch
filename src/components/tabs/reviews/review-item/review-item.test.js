import React from "react";
import renderer from "react-test-renderer";
import ReviewItem from "./review-item";

describe(`ReviewItem`, () => {
  it(`renders correctly`, () => {
    const dom = <ReviewItem />;
    const component = renderer.create(dom);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
