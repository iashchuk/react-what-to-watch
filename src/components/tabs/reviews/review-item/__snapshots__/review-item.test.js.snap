// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewItem renders correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    />
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      />
      <time
        className="review__date"
      />
    </footer>
  </blockquote>
  <div
    className="review__rating"
  />
</div>
`;
